templates/openharmony/entry/build-profile.json5

```diff
@@ -10,11 +10,12 @@
       ],
       "cppFlags": "",
     },
-    "compileMode": "esmodule",
+    "compileMode": "jsbundle",
   },
   "targets": [
     {
-      "name": "default"
+      "name": "default",
+      "runtimeOS": "HarmonyOS"
```

Comment:
what about the OS is OpenHarmony?

Reply:
I think this should be a bug that is not available to environments using Openharmony. We can only use harmonyos.